tracee-rules/signatures/rego/regosig/traceerego.go

```diff
@@ -31,10 +31,18 @@ const querySelectedEvents string = "data.main.tracee_selected_events"
 const queryMetadata string = "data.main.__rego_metadoc__"
 
 // NewRegoSignature creates a new RegoSignature with the provided rego code string
-func NewRegoSignature(regoCode string) (types.Signature, error) {
+func NewRegoSignature(regoCodes ...string) (types.Signature, error) {
 	var err error
 	res := RegoSignature{}
-	res.compiledRego, err = ast.CompileModules(map[string]string{"sig": regoCode})
+
+	if len(regoCodes) == 2 {
+		res.compiledRego, err = ast.CompileModules(map[string]string{"sig": regoCodes[0], "helpers": regoCodes[1]})
+	} else if len(regoCodes) == 1 {
+		res.compiledRego, err = ast.CompileModules(map[string]string{"sig": regoCodes[0]})
+	} else {
+		return nil, fmt.Errorf("invalid number of rego modules")
+	}
```

Comment:
you took the effort to make the function variadic but you still use it as named. you should just add all of the provided modules